components/event-header.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="event-header">
+    <h1 class="event-header__name">
+      <span class="event-header__location">{{ location }}</span>
+      {{ name }}
+    </h1>
+    <nav class="language-switch">
+      <ul class="language-switch__list">
+        <li v-for="{ locale } in _allNameLocales" :key="locale">
+          <nuxt-link
+            class="language-switch__link"
+            :class="{ 'language-switch__link--active': route.params.language === locale }"
+            rel="alternate"
+            :hreflang="locale"
+            :lang="locale"
+            :to="{ name: route.name, params: { ...route.params, language: locale } }"
+          >
+            {{ getNativeName(locale) }}
+          </nuxt-link>
+        </li>
+      </ul>
+    </nav>
+  </div>
+</template>
+
+<script>
+  import ISO6391 from 'iso-639-1';
+
+  export default {
+    methods: {
+      getNativeName: ISO6391.getNativeName,
+    },
+    props: {
+      name: String,
+      location: String,
+      _allNameLocales: Array,
+    },
+    data({ $route }) {
+      return { route: $route };
+    },
+  };
+</script>
+
+<style>
+  .event-header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem;
+    background-color: var(--ui--black);
+  }
+
+  .event-header__name {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    font-size: 1rem;
+  }
+
+  .event-header__location {
+    display: block;
+    color: var(--yl-gn-bu--2-5);
```

Comment:
Maybe when PR https://github.com/Deltares/wwa/pull/158 is merged, change the variables and clean up the `colors.css` file some more.

Reply:
3e69fc2